src/main/java/in/twizmwaz/cardinal/module/modules/hill/RegionSave.java

```diff
@@ -0,0 +1,53 @@
+package in.twizmwaz.cardinal.module.modules.hill;
+
+import in.twizmwaz.cardinal.GameHandler;
+import in.twizmwaz.cardinal.module.modules.regions.RegionModule;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.util.BlockVector;
+import org.bukkit.util.Vector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RegionSave {
+
+    private final BlockVector min;
+    private final BlockVector size;
+    private final List<Pair<Material,Byte>> blocks;
+
+    RegionSave(RegionModule region) {
```

Comment:
twiz, this might be used for renewables in the future, that's why i didn't really add a package